Containers/alpine/Dockerfile

```diff
@@ -0,0 +1,5 @@
+# syntax=docker/dockerfile:latest
+FROM alpine:3.21
```

Comment:
(I know unrelated to this PR, but maybe we should change all alpine:3.21.3 base images to alpine:3.21, since we run apk upgrade in any alpine Dockerfile so it will use the latest patch in any case)

Reply:
Good idea! Lets do this in a follow-up :)